src/amo/constants.js

```diff
@@ -21,3 +21,5 @@ export const LANDING_PAGE_THEME_COUNT = 3;
 export const LANDING_PAGE_PROMOTED_EXTENSION_COUNT = 6;
 
 export const DOWNLOAD_FIREFOX_BASE_URL = 'https://www.mozilla.org/firefox/new/';
+export const PROMOTED_ADDONS_SUMO_URL =
+  'https://support.mozilla.org/kb/add-on-badges';
```

Comment:
could just add the utm parameters to the constant here I suppose?

Reply:
The `utm_content` differs on some of these links, but we could potentially define the `utm_source` and `utm_referral` here. I figured it made more sense to just have this be a URL without a querystring, rather than a URL with a partial querystring.